uuidv8_test.go

```diff
@@ -561,3 +561,56 @@ func TestNewWithParams_MaxValues(t *testing.T) {
 		t.Errorf("Generated UUID with max values is invalid: %s", uuid)
 	}
 }
+
+func TestFromString_InvalidInputs(t *testing.T) {
+	tests := []string{
+		"123",                                   // Too short
+		"123e4567e89b12d3a4564266141740000000",  // Too long
+		"123e4567e89b12d3a45642661417400g",      // Invalid character
+		"123e-4567-e89b-12d3-a456-426614174000", // Misplaced dashes
```

Comment:
Please add:
- a valid uid with the dashes at the right place
- an invalid uid with the dash at the right place plus one randomly placed,or simply 36 dashes

Reply:
Sure thing! Added in the latest commit.